apps/backend/src/services/slackService.ts

```diff
@@ -42,6 +48,8 @@ const getServicePath = (service: SlackService): string | null => {
       return process.env.SLACK_GROUP_SESSION || null;
     case SlackService.FEEDBACK:
       return process.env.SLACK_FEEDBACK || null;
+    case SlackService.STRAVA_UPLOAD:
+      return process.env.SLACK_STRAVA_UPLOAD || null;
```

Comment:
I've tried to strip down the usage of these – so just use the general one, if we end up doing this on the server

Reply:
Done ✅